main.go

```diff
@@ -157,6 +158,56 @@ func main() {
 				return err
 			}
 
+			for _, env := range repo.Environments {
+				var reviewerIds []int
+				for _, username := range env.Reviewers {
+
+					user, err := github.GetUser(ctx, &github.GetUserArgs{Username: username})
+					if err != nil {
+						return err
+					}
+					userId, err := strconv.Atoi(user.Id)
```

Comment:
this seems fishy to me -- why is the user.Id a string but then RepositoryEnvironmentReviewerArray takes an int array as argument?

Reply:
Maybe a difference between the GitHub interface returning strings but pulumi's API assuming IDs are ints?